tests/integration/conftest.py

```diff
@@ -92,7 +88,14 @@ def f(hugr: Package, expected: Any, fn_name: str = "main"):
 
 @pytest.fixture
 def run_int_fn():
-    return _run_fn("run_int_function")
+    run_to_res = _run_fn("run_int_function")
```

Comment:
The awkward thing is that this makes `run_int_fn` (the same as before but) different from `run_float_fn`, so perhaps they should be named differently. `check_int_fn` perhaps, although it'd be less change to rename the new `run_float_fn` instead....thoughts?